Extension/Javascript-Extension/Visual-Studio-Code/code-snippet.md

```diff
@@ -34,20 +34,20 @@ The following steps help you to use the Syncfusion Angular code snippet in your
 
 5. After adding the Syncfusion Angular component to the html file, use the tab key to fill in the required values to render the component with data. You can also find the Syncfusion help link at the top of the added snippet to learn more about the new Syncfusion Angular component feature.
 
-    ![Help](images/Help.png)
+    ![Help in JavaScript Extension](code-snippet_images/javascript-extension-help.png)
 
 ## Configure Angular application with Syncfusion
 
 The Syncfusion Angular snippet only add the code snippet alone in the html file. You need to configure the Angular application with Syncfusion by adding the required Syncfusion Angular NPM, component modules, and themes by manually. To configure, refer the steps below:
 
 1. Open the Angular package.json file and add the required Syncfusion Angular individual NPM package(s) for the Syncfusion Angular components manually. Then navigate to the packages.json file location in the command prompt and run the ***npm install*** command to restore all the Syncfusion Angular NPM packages.
 
-    ![NPM Package](images/NPM.png)
+    ![NPM Package in JavaScript Extension](code-snippet_images/javascript-extension-npm-package.png)
```

Comment:
Check and correct the Image Folder name in all places.

Reply:
corrected the image folder name